src/main/java/cheese/command/ByeCommand.java

```diff
@@ -0,0 +1,30 @@
+package cheese.command;
+
+import cheese.data.TaskList;
+import cheese.storage.Storage;
+import cheese.ui.Ui;
+
+/**
+ * Represents a user command to exit the program.
+ */
+public class ByeCommand extends Command {
+    /**
+     * Executes operation to display goodbye message.
+     * 
+     * @param {@inheritDoc}
+     */
+    @Override
+    public void execute(TaskList taskList, Storage storage, Ui ui) {
+        ui.showGoodbye();
```

Comment:
Something that could be done, is that since it seems that you are only using `ui` to print certain strings, maybe you could use `ui` as a static class, that way you don't need to keep parsing it in as a variable, you could just do `Ui.showGoodbye()`

Reply:
that's a good idea! I'll refactor this for the Gui